CORRECTIONS_URGENTES_NOV_19_2025.md

```
@@ -0,0 +1,281 @@
+
+2. **`f199264`** - chore: ajout lien symbolique pour le répertoire public du build
+   - Création du lien symbolique server/_core/public -> ../../dist/public
+
+3. **`1086ff4`** - fix: correction URL base de données PostgreSQL
+   - Correction de l'URL de connexion PostgreSQL (pooler correct)
+   - Ajout des scripts de test de connexion DB
+   - Vérification des utilisateurs et tables existants
+
+## 🔗 Pull Request
+
+**URL:** https://github.com/doriansarry47-creator/planning/pull/15
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)